dashboard/app/models/school_info.rb

```diff
@@ -32,6 +32,7 @@ class SchoolInfo < ActiveRecord::Base
     SCHOOL_TYPE_PUBLIC = "public".freeze,
     SCHOOL_TYPE_HOMESCHOOL = "homeschool".freeze,
     SCHOOL_TYPE_AFTER_SCHOOL = "afterschool".freeze,
+    SCHOOL_TYPE_ORGANIZATION = "organization".freeze,
```

Comment:
Given that this type was missing, should we have some validation that the type passed in is from this list? That makes sense even when validation_type is set to `VALIDATION_NONE`.

Reply:
Updated [in this commit](https://github.com/code-dot-org/code-dot-org/pull/20539/commits/8243b52e302085c4ae5fec1c18fa6c2234754572), although I'm not sure I know the full ramifications of this.  What do we do if this causes some existing records to no longer validate?

Reply:
Here are the types I see in prod:
```
select distinct(school_type) from school_infos;
+--------------+
| school_type  |
+--------------+
| public       |
| charter      |
| other        |
| private      |
| NULL         |
| homeschool   |
| afterschool  |
| organization |
+--------------+
8 rows in set (0.32 sec)
```